jobs/pacman-awsrules/src/main/java/com/tmobile/cloud/gcprules/APIKeys/ConfigureApiRestriction.java

```diff
@@ -87,7 +87,10 @@ private boolean checkForRestrictAPI(String vmEsURL, Map<String, Object> mustFilt
             logger.info("hit array size {}",hitsJsonArray.size());
             JsonObject apiKeys = (JsonObject) ((JsonObject) hitsJsonArray.get(0))
                     .get(PacmanRuleConstants.SOURCE);
-
+            //handling "restrictions" attribute being NULL
```

Comment:
Remove this comment, code is self descriptive enough

Reply:
@mkinchin-paladin Removed this in latest push